### PR TITLE
Add space delete command

### PR DESCRIPTION
**Pull request by dborsatto**
## Summary

Add `contentful space delete` command

## Todos

- [x] Implemented feature
- [x] Add tests

### CODE REVIEW

File: lib/cmds/space_cmds/delete.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+export const command = 'delete'
+export const desc = 'Deletes a space'
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful space delete')
+    .option('space-id', {
+      alias: 's',
+      describe: 'ID of the Space to seed the data to',
```
Comment (loudmouth) [inline lib/cmds/space_cmds/delete.js]: should be `'ID of the Space to delete'`

File: lib/cmds/space_cmds/delete.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    feature: argv.feature || 'space-seed'
+  })
+
+  const space = await client.getSpace(spaceId)
+
+  if (!argv.yes) {
+    log(`You are about to delete your ${highlightStyle(space.name)} (${highlightStyle(space.sys.id)}) space.`)
+    log()
+
+    const confirm = await confirmation('Do you want to apply the changes to your space now?')
```
Comment (loudmouth) [inline lib/cmds/space_cmds/delete.js]: Maybe we should say: `'Are you sure you want to delete your space and all it's content? This cannot be undone'`

File: lib/cmds/space_cmds/delete.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    log()
+
+    if (!confirm) {
+      log(warningStyle(`User aborted the deletion of the space.`))
```
Comment (loudmouth) [inline lib/cmds/space_cmds/delete.js]: Referring to the user who is currently using the thing in third person here sounds strange.

maybe `'Space deletion aborted'`.

File: lib/cmds/space_cmds/delete.js (hunk 1 above)
Comment (dborsatto) [reply lib/cmds/space_cmds/delete.js]: Copy/paste leftover 🤦‍♂️

File: lib/cmds/space_cmds/delete.js (hunk 2 above)
Comment (dborsatto) [reply lib/cmds/space_cmds/delete.js]: *`its` :trollface:

**Pull request merged**